File: src/main/scala/net/machinemuse/powersuits/item/ItemElectricArmor.scala
package net.machinemuse.powersuits.item

import net.machinemuse.api.electricity._
import net.minecraft.inventory.EntityEquipmentSlot
import net.minecraft.item.{ItemArmor, ItemStack}

abstract class ItemElectricArmor(material: ItemArmor.ArmorMaterial, index1: Int, index2: EntityEquipmentSlot)
  extends ItemArmor(material, index1, index2)
  with ModularItemBase
  with MuseElectricItem {
  override def getToolTip(stack: ItemStack): String = null
}

File: src/main/java/net/machinemuse/powersuits/item/armor/ItemElectricArmor.java
package net.machinemuse.powersuits.item.armor;

import net.minecraft.inventory.EntityEquipmentSlot;
import net.minecraft.item.ItemArmor;

/**
 * Power handling base class for armor
 */
public class ItemElectricArmor extends ItemArmor {
    public ItemElectricArmor(EntityEquipmentSlot slots, Properties builder) {
        super(MPSArmorMaterial.EMPTY_ARMOR, slots, builder);
    }

//    @Nullable
//    @Override
//    public ICapabilityProvider initCapabilities(ItemStack stack, @Nullable NBTTagCompound nbt) {
//        return new MPSCapProvider(stack);
//    }
}

